qubes_config/global_config.glade

```diff
@@ -814,7 +814,95 @@
+                                <property name="wrap">True</property>
+                                <property name="xalign">0</property>
+                                <style>
+                                  <class name="main_text"/>
+                                </style>
+                              </object>
+                              <packing>
+                                <property name="expand">False</property>
+                                <property name="fill">True</property>
+                                <property name="position">0</property>
+                              </packing>
+                            </child>
+                            <child>
+                              <object class="GtkLabel">
+                                <property name="visible">True</property>
+                                <property name="can-focus">False</property>
+                                <property name="label" translatable="yes">Number of disposable qubes to preload from the default disposable template.</property>
+                                <property name="wrap">True</property>
+                                <property name="xalign">0</property>
+                                <style>
+                                  <class name="explanation"/>
+                                </style>
+                              </object>
+                              <packing>
+                                <property name="expand">False</property>
+                                <property name="fill">True</property>
+                                <property name="position">1</property>
+                              </packing>
+                            </child>
+                          </object>
+                          <packing>
+                            <property name="left-attach">0</property>
+                            <property name="top-attach">8</property>
+                          </packing>
+                        </child>
+                        <child>
+                          <object class="GtkBox">
+                            <property name="visible">True</property>
+                            <property name="can-focus">False</property>
+                            <child>
+                              <object class="GtkSpinButton" id="basics_preload_dispvm">
+                                <property name="visible">True</property>
+                                <property name="can-focus">True</property>
+                                <property name="max-length">2</property>
+                                <property name="max-width-chars">2</property>
+                              </object>
+                              <packing>
+                                <property name="expand">False</property>
+                                <property name="fill">True</property>
```

Comment:
I think this is the line that causes the spinbutton to be a bit too big ( 
![image](https://github.com/user-attachments/assets/3755413d-4578-4b96-8e8f-9d9ddff9ed3b)
); It should not bee filling the entire space, it should have natural size (same as comboboxes above), aligned to the vertical center.

Reply:
Also, the left border of the spinbutton is not aligned with the comboboxes - it should be vertically aligned, probably the margin is wrong somewhere